### ISSUE TITLE
Audio stream not working on Firefox Android

### ISSUE THREAD

**Issue by bugz313** [goldfire/howler.js#1159]
Firefox for Android version 66.0.2

I am trying to play a radio stream but it's stuck loading forever. Static sound files play without problems.

I also tried playing the Live Radio demos from https://howlerjs.com/ and same thing happens.

**Comment by darren-dev** [comment]
Without code examples, output logs, or anything like that, no one will be able to assist you correctly.

**Comment by bugz313** [comment]
> Without code examples, output logs, or anything like that, no one will be able to assist you correctly.

You can just visit howlerjs.com in Firefox on Android, scroll down to Demos, open Live Radio demos and try to play BBC Radio 1 or any other live radio. It never plays.

For code you can check: examples/radio/radio.js

**Comment by darren-dev** [comment]
> Firefox for Android version 66.0.2

Yes, however I may not have the EXACT system you are using. I personally have a Note 4 (Which is older) and things work fine. Providing code isn't to prove that there is an issue, it's so another developer can look and try provide assistance.

**Comment by goldfire** [comment]
Are you still seeing this issue? I've tested on the latest Firefox on 2 different Android devices and the live radio demo worked on both.

**Comment by roks0n** [comment]
I'm experiencing the same issue on my Android. Howler's live demos don't work either. I'm not seeing any errors. 

When I was further debugging on my own site, I've noticed the following:
- Howler instance is stuck on `loading` state
- `canplaythrough` even is triggered after I `unload` the instance (gets triggered when I press STOP button)
- instance was able to get into the `loaded` at random times which occurred very rarely (from the top of my head 5% of the times). When that occurred I unloaded the instance and tried playing it again which resulted in the same issue as before
- I've was also listening to all events on the howler instance, but they didn't get triggered at any point (see the list of methods in the sample code bellow) 

This is how I initialize the howler instance:
```
new Howl({
    src: ['https://strm.radionula.com/classics'],
    html5: true,
    format: ['mp3'],
    onload: () => console.log('onload'),
    onloaderror: (e, msg) => console.log('onloaderror', e, msg),
    onplayerror: (e, msg) => console.log('onplayerror', e, msg),
    onplay: () => console.log('onplay'),
    onend: () => console.log('onend'),
    onpause: () => console.log('onpause'),
    onrate: () => console.log('onrate'),
    onstop: () => console.log('onstop'),
    onseek: () => console.log('onseek'),
    onfade: () => console.log('onfade'),
    onunlock: () => console.log('onunlock'),
});
```

Firefox for Android version 66.0.2.
Android v9 (OxygenOS v9.0.13).

I've also tested on :
- works OK on desktop FF/Chrome
- works OK on FF/Safari on iOS 12
- works OK on Chrome on Android (same specs as above)

Let me know if this information is sufficient. I'll try to do further debugging and will prepare a small project that will replicate the issue. Until then, if anyone has any ideas/suggestions what would be worth checking I'd greatly appreciate it.

**Comment by roks0n** [comment]
I haven't made much progress yet, but this is the sample project with which I'm able to replicate the issue: https://github.com/roks0n/howler-demo

Note:
- I've added [some custom event listeners to howler.js file](https://github.com/roks0n/howler-demo/blob/master/howler.js#L2213-L2223) for debugging purposes

**Comment by bugz313** [comment]
@goldfire I am still seeing the issue. Could this issue be reopened please?

**Comment by roks0n** [comment]
Just to give my update: nothing changed since I posted and I haven't made any progress on the issue. I'll keep everyone updated if anything changes.

**Comment by eladkolet** [comment]
we experience the same issues. any update on this will be great.

**Comment by bugz313** [comment]
@eladkolet @roks0n The issue on our side was caused by the stream not being secure and using a port. Now it works.

**Comment by nikhilbhalwankar** [comment]
Still facing the issue on firefox for android. Works well on Chrome but not on Firefox. We already have https url for the audio.

Mobile :- Nokia 6.1 Plus
Firefox for Android version:- 68.3

**Comment by iamawebgeek** [comment]
the same problem. gets stuck on loading state for some random sounds on firefox only, but works fine for the rest. anyone was able to figure out how to fix this?
